### PR TITLE
Damaris Landers adds Manchester, UK

### LINKED ISSUE #36: Damaris-Landers follows GitHub Flow
# Manchester, UK

- [x] create a **branch**
- [ ] add some **commits** - snapshots of your project
- [ ] open *pull request* - conversation about code
- [ ] collaborate - discuss and review
- [ ] merge with `master branch` upon approval

### PR DESCRIPTION
**Pull request by Damaris-Landers**
This Pull Request is to add Manchester, UK to the list of hometowns.

@daveearley @pH-7 users chosen to mention at random 😄 

This Pull Request closes #36.

### CODE REVIEW

File: Damaris-Landers-ManchesterUK.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Manchester, UK
+
+### Restaurants
+
+- Black Dog Ballroom
```
Comment (juan-tex) [inline Damaris-Landers-ManchesterUK.md]: @Damaris-Landers  I'll check it next time I pass by Manchester. Does Ballroom means there is dancing as well?

File: Damaris-Landers-ManchesterUK.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+- The Old Rectory
+
+### Attractions / Places to see
+
+- Arndale Centre
+- University of Manchester
+
+### Fun Nearby
```
Comment (juan-tex) [inline Damaris-Landers-ManchesterUK.md]: @Damaris-Landers  New comment, as I pressed "Approve" before.

File: Damaris-Landers-ManchesterUK.md (hunk 1 above)
Comment (Damaris-Landers) [inline Damaris-Landers-ManchesterUK.md]: Yes! Lots of dancing but not Ballroom Dancing (as you might think).

Comment (juan-tex) [inline Damaris-Landers-ManchesterUK.md]: Always on the look for Argentine tango places!

File: Damaris-Landers-ManchesterUK.md (hunk 2 above)
Comment (Damaris-Landers) [inline Damaris-Landers-ManchesterUK.md]: Ok. Noted.

**Pull request merged**